tests/test.tex

```diff
@@ -23,6 +23,8 @@
 
 \begin{Spoiler}
 Voici un secret
+
+... Sur plusieurs lignes
```

Comment:
can you add a external link inside a spoiler please? In some contents it resulted in non clickable links.

Reply:
I confirm that this is now clickable, and seems to generate no problem :)